virtac/atip_ioc_entry.py

```diff
@@ -32,15 +32,25 @@ def main():
                 value = caget("SR-CS-RING-01:MODE", format=2)
                 ring_mode = value.enums[int(value)]
             except ca_nothing:
-                ring_mode = "DIAD"
+                ring_mode = "I04"
 
     # Create PVs.
-    server = atip_server.ATIPServer(
+    if sys.argv[2:]:
+        server = atip_server.ATIPServer(
+            ring_mode,
+            DATADIR / "limits.csv",
+            DATADIR / "feedback.csv",
+            DATADIR / "mirrored.csv",
+            DATADIR / "tunefb.csv",
+            sys.argv[2]
```

Comment:
Unless I'm mistaken, this will pass any string straight down, before it eventually gets evaluated as True or False. That doesn't make much logical sense, and breaks any type checking you want to add.

This is a strong argument to make the (simple) switch to argparse.